io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/reader/XLSInstanceReader.java

```diff
@@ -84,8 +84,7 @@ public boolean isCancelable() {
 	protected IOReport execute(ProgressIndicator progress, IOReporter reporter)
 			throws IOProviderConfigurationException, IOException {
 
-		boolean skipFirst = getParameter(CommonSchemaConstants.PARAM_SKIP_FIRST_LINE).as(
-				Boolean.class);
+		int skip_N_lines = getParameter(CommonSchemaConstants.PARAM_SKIP_N_LINES).as(Integer.class);
```

Comment:
Same as mentioned previously. We need to take into account that the parameter is used in the old way and be backwards compatible.